starter_code/queries.md

```diff
@@ -4,23 +4,30 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+{name : "Babelgum"}
+{name : 1}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+{number_of_employees: {$gt: 5000}}
+{number_of_employees: 1}
+limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fileds.
 
-<!-- Your Code Goes Here -->
+{$and: [ {founded_year: {$gte: 2000}}, {founded_year: {$lte: 2005}} ]}
```

Comment:
To make this shorter, you could do this: {founded_year: {$gte: 2000, $lte: 2005 }}